### ISSUE TITLE
Get DSCOVR image at current time

### ISSUE THREAD

**Issue by TheNeuralBit** [domoritz/himawari-8-chrome#4]
It looks like an entire days-worth of DSCOVR images are dumped at once, delayed by a day or two. So the "latest" image is typically just the last image for that day. It'd be nice if there were an option to grab the most recent image close to the current time so the image changes throughout the day, like the himawari images.

I'd be willing to contribute this change if its something you're interested in.

**Comment by domoritz** [comment]
Yeah, sadly images are dumped once a day. I like the idea of having a slightly larger delay but have images change throughout the day. However, I think users should have the option to disable this behavior in the settings.

**Comment by erget** [comment]
For me DSCOVR images aren't downloaded. Did the web service change?

**Comment by domoritz** [comment]
@erget Works for me. Can you file a bug with the steps you went through? 

<img width="900" alt="screen shot 2018-06-18 at 08 19 54" src="https://user-images.githubusercontent.com/589034/41545624-749c3d70-72d0-11e8-8fde-211300e33854.png">

**Comment by erget** [comment]
Seems to have resolved itself - maybe I was having problems due to the spotty connection (I've been at airports all day today). Now that I'm in my hotel I get the DSCOVR imagery just fine.
